### ISSUE TITLE
Velocity tracking in gazebo

### ISSUE THREAD

**Issue by maxspahn** [INTERACT-tud-amr/dinova#2]
Velocity tracking is poor for joint 2 in the simulation.
As soon as the arm starts dropping a bit, a reference velocity can no longer be tracked.

**Comment by TomasMerva** [comment]
Yes, the velocity interface only works when the gravity in Gazebo is turned off. Don't know how to fix it.

**Comment by TomasMerva** [comment]
For now, the gravity is turned off by default for `dinova` and `kinova` with the commit: 930a34a0e2fbea1f23b552ed587599a8a702e8a1
